larq/optimizers_v1.py

```diff
@@ -0,0 +1,180 @@
+import tensorflow as tf
+import numpy as np
+import larq as lq
+
+from larq import utils
+from copy import deepcopy
+
+from tensorflow.python.keras.optimizers import *
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import random_ops
+from tensorflow.python.keras import backend as K
```

Comment:
We should try not to import from `tensorflow.python` as best as we can since those imports might changes (and `*` might give us very weird behaviour)

Reply:
fixed it.